src/utils/users.client.ts

```diff
@@ -0,0 +1,50 @@
+import grpc from "grpc";
+import config from "../config";
+import { loadSync } from "@grpc/proto-loader";
+import { IUser } from "./users.interface";
+
+const PROTO_PATH = `${__dirname}/../../proto/users.proto`;
```

Comment:
why this syntax?

Reply:
Please add a suggestion because it was in every example.

Reply:
`protoPath`?
Just a simple camel-case